### PR TITLE
Log admin respawns

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Title

## Changelog
:cl:
admin: Bypassing respawn as an admin will leave a trace in admin logs
/:cl:

### CODE REVIEW

**Comment by Dolth** [comment]
I thought it already did lol.

**Review by TiviPlus — CHANGES_REQUESTED** [review]
use key_name

File: code/modules/mob/living/carbon/xenomorph/hive_datum.dm [hunk 1]
```diff
@@ -641,6 +642,7 @@ to_chat will check for valid clients itself already so no need to double check f
 	if(!isnewplayer(xeno_candidate) && XENODEATHTIME_CHECK(xeno_candidate))
 		if(check_other_rights(xeno_candidate.client, R_ADMIN, FALSE))
 			if(tgui_alert(xeno_candidate, "You wouldn't normally qualify for this respawn. Are you sure you want to bypass it with your admin powers?", "Bypass Respawn", list("Yes", "No")) != "Yes")
+				log_admin("[xeno_candidate.client.ckey] used his admin power to bypass respawn before his timer was over")
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/hive_datum.dm:645]:
```suggestion
				log_admin("[key_name(xeno_candidate)] used his admin power to bypass respawn before his timer was over")
```

File: code/modules/mob/mob_verbs.dm [hunk 2]
```diff
@@ -61,6 +61,7 @@
 	if(DEATHTIME_CHECK(usr))
 		if(check_other_rights(usr.client, R_ADMIN, FALSE))
 			if(tgui_alert(usr, "You wouldn't normally qualify for this respawn. Are you sure you want to bypass it with your admin powers?", "Bypass Respawn", list("Yes", "No"), 0) != "Yes")
+				log_admin("[key_name(xeno_candidate)] used his admin power to bypass respawn before his timer was over")
```
Comment (BraveMole) [inline code/modules/mob/mob_verbs.dm:64]:
```suggestion
				log_admin("[key_name(usr)] used his admin power to bypass respawn before his timer was over")
```

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**